feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/vif.c

```diff
@@ -803,9 +806,10 @@ bool vif_state_update(struct uci_section *s, struct schema_Wifi_VIF_Config *vcon
 	}
 	vif_state_security_get(&vstate, tb);
 	vif_state_custom_options_get(&vstate, tb);
-	vif_state_captive_portal_options_get(&vstate, s);
-	vif_state_dhcp_allowlist_get(&vstate);
-
+	if(!strcmp(network_name,"lan"))
+		vif_state_captive_portal_options_get(&vstate);
+	if(!strcmp(network_name,"lan"))
```

Comment:
Why did the dhcp-allowlist get pushed into this "if" statement at line 811?  If it needs to be in the "if", then the "if" at 809 is identical and. should be used for both.

Reply:
we have captive_allowlist and captive_portal, two separated entires in the vif config and vif state table. When that respective entry changed, then only I'm trying to check the condition and asking to update the state

Reply:
So you are saying it is correct for them both to depend on "network_name==lan".  Correct?
If the condition is the same, then don't test it twice;  line 810 and 812 should be in the same if statement at line 809.

Reply:
yes, I'll quickly change and update